bioptim/dynamics/dynamics_functions.py

```diff
@@ -124,7 +128,27 @@ def torque_driven(
         if fatigue is not None and "tau" in fatigue:
             dxdt = fatigue["tau"].dynamics(dxdt, nlp, states, controls)
 
-        return dxdt
+        if with_contact or fatigue is not None:
+            defects = MX.zeros(1)
```

Comment:
How come do we skip everything if with_contact or fatigue is True?

Reply:
I need to manage contact once PR #426 will be accepted and I just didn't figure out how to manage fatigue. :)